tools/gltfViewer/README.md

```diff
@@ -0,0 +1,39 @@
+# Simple glTL Viewer
+
+The gltfViewer application can be use to previous glTF files, including
```

Comment:
"preview"  or "view" instead of "previous"?

Reply:
done